packages/api/tests/es/test-queries.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const test = require('ava');
+
+const { randomId } = require('@cumulus/common/test-utils');
+
+const indexer = require('../../es/indexer');
+const { Search } = require('../../es/search');
+const { fakeGranuleFactoryV2 } = require('../../lib/testUtils');
+const { bootstrapElasticSearch } = require('../../lambdas/bootstrap');
+
+const granules = [
+  fakeGranuleFactoryV2(),
+  fakeGranuleFactoryV2({ granuleId: randomId('granprefix') }),
+  fakeGranuleFactoryV2({ granuleId: randomId('granprefix'), status: 'failed' }),
+  fakeGranuleFactoryV2({ status: 'failed' })
+];
+
+let esClient;
+const esIndex = randomId('esindex');
+const esAlias = randomId('esalias');
+process.env.ES_INDEX = esAlias;
+
+test.before(async () => {
+  // create the elasticsearch index and add mapping
+  await bootstrapElasticSearch('fakehost', esIndex, esAlias);
+  esClient = await Search.es();
+
+  await Promise.all(
+    granules.map((granule) => indexer.indexGranule(esClient, granule, esAlias))
+  );
+});
+
+test.after.always(async () => {
+  await esClient.indices.delete({ index: esIndex });
+});
+
+test('Search with prefix returns correct granules', async (t) => {
```

Comment:
Can you add a test using both prefix and infix please?

Reply:
Thanks.  Added